src/views/Settings.vue

```diff
@@ -8,68 +8,168 @@
     </ion-header>
     
     <ion-content :fullscreen="true">
-      <!-- OMS information -->
-      <ion-item>
-        <ion-icon :icon="codeWorkingOutline" slot="start"/>
-        <ion-label>{{ $t("OMS") }}</ion-label>
-        <p slot="end">{{ baseURL ? baseURL : instanceUrl }}</p>
-      </ion-item>
-      <!-- Time zone -->
-      <ion-item>
-        <ion-icon :icon="timeOutline" slot="start"/>
-        <ion-label> {{ userProfile && userProfile.userTimeZone ? userProfile.userTimeZone : '-' }} </ion-label>
-        <ion-button @click="changeTimeZone()" slot="end" fill="outline" color="dark">{{ $t("Change") }}</ion-button>
-      </ion-item>
-      <!-- Profile of user logged in -->
-      <ion-item>
-        <ion-icon :icon="personCircleOutline" slot="start" />
-        <ion-label>{{ userProfile !== null ? userProfile.partyName : '' }}</ion-label>
-        <ion-button slot="end" fill="outline" color="dark" @click="logout()">{{ $t("Logout") }}</ion-button>
-      </ion-item>
+      <div class="user-profile">
+        <ion-card>
+          <ion-item lines="full">
+            <ion-avatar slot="start" v-if="userProfile?.partyImageUrl">
+              <Image :src="userProfile?.partyImageUrl"/>
```

Comment:
I think optional chaining is not needed as we are already checking partyImageUrl in v-if condition.
```suggestion
              <Image :src="userProfile.partyImageUrl"/>
```

Reply:
Made the requested change.